retrofit/src/main/java/retrofit/BuiltInConverters.java

```diff
@@ -25,21 +25,40 @@
 import static retrofit.Utils.closeQuietly;
 
 final class BuiltInConverters extends Converter.Factory {
+  private volatile OkHttpResponseBodyConverter streamingResponseBodyConverter;
```

Comment:
Why volatile ? it doesn't guarantee you single-instances, and it harms your performance